packages/jsii-pacmak/lib/targets/dotnet/filegenerator.ts

```diff
@@ -50,7 +50,7 @@ export class FileGenerator {
     const rootNode = xmlbuilder.create('Project', { encoding: 'UTF-8', headless: true });
     rootNode.att('Sdk', 'Microsoft.NET.Sdk');
     const propertyGroup = rootNode.ele('PropertyGroup');
-    propertyGroup.ele('TargetFramework', 'netcoreapp2.1');
+    propertyGroup.ele('TargetFrameworks', 'netcoreapp2.1;netcoreapp3.0');
```

Comment:
Uh. Wut? The tag name changes?

Reply:
Yeah - there's a tag for singular target, and one for multi-targeting. Thank you (not) msbuild.